app/build.gradle.kts

```diff
@@ -130,6 +142,15 @@ kotlin {
     }
 }
 
+configurations.all {
+    resolutionStrategy.dependencySubstitution {
+        if ((userProperties["localSdk"] as String?).toBoolean()) {
```

Comment:
Would these checks work as booleans directly?

```kotlin
if ((userProperties["localSdk"] as Boolean?) == true) {
```

Reply:
That was my initial attempt, unfortunately you get a cast error, string can't be cast to class boolean.

Reply:
Interesting, ok. Thanks for checking!